src/main/scala/org/camunda/feel/impl/interpreter/FeelInterpreter.scala

```diff
@@ -18,101 +18,11 @@ package org.camunda.feel.impl.interpreter
 
 import java.time.{Duration, Period}
 import org.camunda.feel.FeelEngine.UnaryTests
+import org.camunda.feel.api.EvaluationFailureType
 import org.camunda.feel.context.Context
 import org.camunda.feel.valuemapper.ValueMapper
-import org.camunda.feel.syntaxtree.{
-  Addition,
-  ArithmeticNegation,
-  AtLeastOne,
-  ClosedConstRangeBoundary,
-  ClosedRangeBoundary,
-  Comparison,
-  Conjunction,
-  ConstBool,
-  ConstContext,
-  ConstDate,
-  ConstDateTime,
-  ConstDayTimeDuration,
-  ConstInputValue,
-  ConstList,
-  ConstLocalDateTime,
-  ConstLocalTime,
-  ConstNull,
-  ConstNumber,
-  ConstRange,
-  ConstRangeBoundary,
-  ConstString,
-  ConstTime,
-  ConstYearMonthDuration,
-  Disjunction,
-  Division,
-  Equal,
-  EveryItem,
-  Exp,
-  Exponentiation,
-  Filter,
-  For,
-  FunctionDefinition,
-  FunctionInvocation,
-  FunctionParameters,
-  GreaterOrEqual,
-  GreaterThan,
-  If,
-  In,
-  InputEqualTo,
-  InputGreaterOrEqual,
-  InputGreaterThan,
-  InputInRange,
-  InputLessOrEqual,
-  InputLessThan,
-  InstanceOf,
-  IterationContext,
-  JavaFunctionInvocation,
-  LessOrEqual,
-  LessThan,
-  Multiplication,
-  NamedFunctionParameters,
-  Not,
-  OpenConstRangeBoundary,
-  OpenRangeBoundary,
-  PathExpression,
-  PositionalFunctionParameters,
-  QualifiedFunctionInvocation,
-  RangeBoundary,
-  Ref,
-  SomeItem,
-  Subtraction,
-  UnaryTestExpression,
-  Val,
-  ValBoolean,
-  ValContext,
-  ValDate,
-  ValDateTime,
-  ValDayTimeDuration,
-  ValError,
-  ValFunction,
-  ValList,
-  ValLocalDateTime,
-  ValLocalTime,
-  ValNull,
-  ValNumber,
-  ValRange,
-  ValString,
-  ValTime,
-  ValYearMonthDuration,
-  ZonedTime
-}
-import org.camunda.feel.{
-  Date,
-  DateTime,
-  DayTimeDuration,
-  LocalDateTime,
-  LocalTime,
-  Number,
-  Time,
-  YearMonthDuration,
-  logger
-}
+import org.camunda.feel.syntaxtree.{Addition, ArithmeticNegation, AtLeastOne, ClosedConstRangeBoundary, ClosedRangeBoundary, Comparison, Conjunction, ConstBool, ConstContext, ConstDate, ConstDateTime, ConstDayTimeDuration, ConstInputValue, ConstList, ConstLocalDateTime, ConstLocalTime, ConstNull, ConstNumber, ConstRange, ConstRangeBoundary, ConstString, ConstTime, ConstYearMonthDuration, Disjunction, Division, Equal, EveryItem, Exp, Exponentiation, Filter, For, FunctionDefinition, FunctionInvocation, FunctionParameters, GreaterOrEqual, GreaterThan, If, In, InputEqualTo, InputGreaterOrEqual, InputGreaterThan, InputInRange, InputLessOrEqual, InputLessThan, InstanceOf, IterationContext, JavaFunctionInvocation, LessOrEqual, LessThan, Multiplication, NamedFunctionParameters, Not, OpenConstRangeBoundary, OpenRangeBoundary, PathExpression, PositionalFunctionParameters, QualifiedFunctionInvocation, RangeBoundary, Ref, SomeItem, Subtraction, UnaryTestExpression, Val, ValBoolean, ValContext, ValDate, ValDateTime, ValDayTimeDuration, ValError, ValFunction, ValList, ValLocalDateTime, ValLocalTime, ValNull, ValNumber, ValRange, ValString, ValTime, ValYearMonthDuration, ZonedTime}
```

Comment:
Just a casual 1186 character line 😆

Reply:
Yes, currently the formatting doesn't work. :sweat_smile: 

I will improve the situation with https://github.com/camunda/feel-scala/issues/654.